tests/filecheck/mlir-conversion/with-mlir/mlir_opt_fail.mlir

```diff
@@ -1,5 +1,6 @@
 // RUN: xdsl-opt %s -p mlir-opt{arguments='--hello','--mlir-print-op-generic'} --print-op-generic --verify-diagnostics | filecheck %s
 // RUN: xdsl-opt %s -p mlir-opt[this-probably-will-never-be-an-MLIR-pass-name] --print-op-generic --verify-diagnostics | filecheck %s
+// RUN: xdsl-opt %s -p mlir-opt{executable=cat} --print-op-generic --verify-diagnostics | filecheck %s
```

Comment:
🤖 🐱

Reply:
Sadly `cat` made the test not fail :/

Reply:
Try `dog`

Reply:
We *need* `cowsays` filechecks somewhere